astropy/io/ascii/tests/test_compressed.py

```diff
@@ -35,3 +35,12 @@ def test_xz(filename):
     t_uncomp = read(get_pkg_data_filename(filename.replace(".xz", "")))
     assert t_comp.dtype.names == t_uncomp.dtype.names
     assert np.all(t_comp.as_array() == t_uncomp.as_array())
+
+
+@pytest.mark.xfail(not HAS_UNCOMPRESSPY, reason="requires uncompresspy")
+@pytest.mark.parametrize("filename", ["data/short.rdb.Z", "data/ipac.dat.Z"])
+def test_lzw(filename):
+    t_comp = read(get_pkg_data_filename(filename))
+    t_uncomp = read(get_pkg_data_filename(filename.replace(".Z", "")))
```

Comment:
We only support python >= 3.11, so can use `filename.removesuffix(".Z")`
(I realize all the other tests do the same... so maybe not worth it)